saltyrtc/client.ts

```diff
@@ -225,7 +225,14 @@ export class Client {
             }
 
             // Send offer
-            this.peerConnection.sendOffer();
+            this.peerConnection.createOffer().then(
+                (offer) => {
+                    this.peerConnection.setLocalDescription(offer); // TODO: Should we handle errors?
```

Comment:
Not sure whether there's something we should do if setting the local description fails. Maybe errors on top level is something that should be done through event handlers?